views/movies/edit.hbs

```diff
@@ -0,0 +1,12 @@
+<h1>Edit a Movie</h1>
+{{#with movieData}}
+  <form action="/movies/edit/{{_id}}" method="POST">
+    <label for="title">Title </label>
+    <input type="text" name="title" id="title" value="{{title}}">
+    <label for="genre">Genre </label>
+    <input type="text" name="genre" id="genre" value="{{genre}}">
+    <label for="plot">Plot </label>
+    <input type="text" name="plot" id="plot" value="{{plot}}">
+    <button type="submit">Edit Movie</button>
```

Comment:
Here you would also need to consider looping on the celebrities and figuring out how to show the ones in the movie cast as selected.